modules/installation-azure-user-defined-routing.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-private.adoc
+
+[id="installation-azure-user-defined-routing_{context}"]
+= User-defined outbound routing
+
+In {product-title}, you can choose your own outbound routing for a cluster to
+connect to the internet. This allows you to skip the creation of public IP
+addresses and public load balancers. This is required if you want to prevent
+exposing external endpoints for a private cluster.
+
+You can configure user-defined routing by modifying parameters in the
+`install-config.yaml` file before installing your cluster. A pre-existing VNet
+is required to use outbound routing when installing a cluster; the installation
+program is not responsible for configuring this.
+
+When configuring a cluster to use user-defined routing, the installation program
+does not create the following resources:
+
+* Outbound rules
+* Kubernetes Loadbalancer
+* Public load balancer configurations for MachineSets
```

Comment:
- Outbound rules for access to Internet.
- Public IPs for the public load balancer.
- Kuberenetes Service object to add the cluster machines to public load balancer backend for outbound.